lib/pagination/adapter.js

```diff
@@ -0,0 +1,35 @@
+const { invalidPageNumber, invalidLimitNumber } = require('./errors');
```

Comment:
You could throw an Error from javascript here, why do you think in this custom errors ?

Reply:
Because I was thinking about custom errors of the Paginator. Right now we just have this 2 basic cases, but in future could be more custom errors strictly related with the package itself, and I thought we should leave a logic to handle specific package errors. Do you suggest to leave this just as `throw 'Invalid Page Number'` instead of having certain specific custom errors that we know our paginator could have?